app/components/navs/nav-rail-pinned-item.jsx

```
@@ -12,42 +12,33 @@ import Link from 'next/link';
+    imgSrc,
+    width,
+    text,
+    selected = false,
+    onClick = () => console.log('No onClick handler defined'),
+    href = '#',
+    extended = false
+}) {
     return (
-        <div className={`nav-item rounded-full cursor-pointer font-medium selected-${props.selected}`} role='link' onClick={props.onClick}>
-            <Link className={`state-layer relative flex extended-gap-${props.extended} items-center justify-between rounded-full padding-${props.extended ? "extended" : "collapsed"}`} href={props.href}>
+        <div className={`nav-item rounded-full cursor-pointer font-medium selected-${selected}`} role='link' onClick={onClick}>
```

Biome (1.9.4)

[error] The HTML element with the interactive role 'link' is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like 'link'.
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)